share/picongpu/tests/CurrentDeposition/lib/python/test/CurrentDeposition/grid_class.py

```diff
@@ -0,0 +1,207 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2023 PIConGPU contributors
+Authors: Hannes Wolf
+License: GPLv3+
+
+Intoduces a grid class. A particle with defined assignment function can do a
+move (also randomly) on the grid from one position the another one. The
+current deposition and current density vector fields can then be calculated.
+"""
+
+
+import numpy as np
+from AfuncW import NGP, CIC, TSC, PSQ, W

```

Comment:
Since you are not importing the array version of these function, why do you need them?

Reply:
removed